apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -122,6 +122,11 @@ function WpcomBlockEditorNavSidebar() {
 		return null;
 	}
 
+	// The `closeUrl` "closes" the editor, returning the user to the dashboard.
```

Comment:
❤️

Reply:
I ended up staying with the `closeUrl` identifier (instead of `backUrl`) because everything refers to this as close. "Back" is probably a better name, but we'd need to rename a whole bunch of filters and other Calypso integration stuff. Hopefully the comment helps a little though.